app/views/users/show.html.erb

```diff
@@ -1,11 +1,19 @@
-<div class="row">
+
```

Comment:
the indentation in this file is all over the place, so it's difficult to see what's going on at a glance.